src/test/groovy/nebula/plugin/publishing/ivy/IvyRemovePlatformDependenciesPluginSpec.groovy

```diff
@@ -192,6 +192,10 @@ dependencies {
                 implementation enforcedPlatform('com.github.sghill.jenkins:jenkins-bom:latest.release')
                 implementation 'test.resolved:a:1.+'
             }
+
+            tasks.withType(GenerateModuleMetadata).configureEach {
+                suppressedValidationErrors.add('enforced-platform')
```

Comment:
This is now required since publishing `enforced-platform` as a dependency in gradle metadata fails. This requires Gradle 7.0 snapshot as Gradle version for this project.